package.json

```diff
@@ -19,6 +19,10 @@
   },
   "engine-strict": true,
   "engineStrict": true,
+  "files": [
+    "*",
+    "src/comm/template/report.html"
+  ],
```

Comment:
I needed to add this so that the template html file gets included on an `npm install`